File: 백준/Bronze/10808. 알파벳 개수/알파벳 개수.py
import sys
input=sys.stdin.readline

def solution(S) :
    alpha = [0] * 26

    for a in S:
        alpha[ord(a) - 97] += 1
        
    print(*alpha)
        
if __name__ == "__main__" :
    S = input().rstrip()
    solution(S)

File: 백준/Bronze/10808. 알파벳 개수/알파벳 개수.js
const fs = require("fs");
const filePath = process.platform === "linux" ? "/dev/stdin" : __dirname + "/input.txt";
const input = fs.readFileSync(filePath).toString().replace(/\r/g, "").trim().split("\n");

let arr = Array(26).fill(0);
function solution(input) {
  input = input[0].split("");

  for (let i = 0; i < input.length; i++) {
    arr[input[i].charCodeAt() - 97]++;
  }

  return [...arr].join(" ");
}

console.log(solution(input));